### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by vadim-kovalyov**
I was a bit confused by the new PR template, so I'm proposing some change to PR template. Please let me know if that make sense.

## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [X] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [X] Title of the pull request is clear and informative.
- [X] Description of the pull request includes a concise summary of the enhancement or bug fix.

### CODE REVIEW

**Review by damonbarry — CHANGES_REQUESTED** [review]
Almost forgot...

You also need to update the `cut_body_after` option in `.github/.kodiak.toml`

File: .github/.kodiak.toml [hunk 1]
```diff
@@ -37,4 +37,4 @@ include_pull_request_author = false # default: false
 strip_html_comments = true # default: false
 
 # remove PR template from merge message body
-cut_body_after = "Azure IoT Edge PR checklist:"
+cut_body_after = "## Azure IoT Edge PR checklist:"
```
[the remark this answers is not in the document]
Comment (vadim-kovalyov) [reply .github/.kodiak.toml:40]: I assumed that people will just remove this line and write descriptions instead :D

[the remark this answers is not in the document]
Comment (vadim-kovalyov) [reply .github/.kodiak.toml:40]: Should I be more specific that people need to `"remove this line and write your PR Description here"`?

[the remark this answers is not in the document]
Comment (damonbarry) [reply .github/.kodiak.toml:40]: Sure. "Please replace this line with your PR description..."

**Pull request merged**